### ISSUE TITLE
./bin/magento di:compile throw error 

### ISSUE THREAD

**Issue by grojanteam** [klevu/metadata#14]
I think so that klevu/productsearch

shouldn't have in older versions metadata as ^1.1 as in older version di:compile throwing error with not existing interface

%message% 0/9 [>---------------------------]   0% < 1 sec 89.0 MiBProxies code generation... 0/9 [>------------------------]   0% < 1 sec 89.0 MiB
Proxies code generation... 1/9 [==>----------------------]  11% < 1 sec 99.0 MiB
Repositories code generation... 1/9 [==>-----------------]  11% < 1 sec 99.0 MiB                                                                               
Interface 'Klevu\Search\Api\Service\Convertor\Sync\Order\ItemDataConvertorInterface' not found In OrderSyncItemDataConvertor.php line 9:... 

can you pls fix this as soon as possible?

#25 9.861   - Locking klevu/module-addtocart (2.7.1)
#25 9.861   - Locking klevu/module-content (2.7.1)
#25 9.861   - Locking klevu/module-frontend-js (1.4.0-beta)
#25 9.861   - Locking klevu/module-logger (1.x-dev 598aa5f)
#25 9.861   - Locking klevu/module-metadata (1.x-dev b25855c)
#25 9.861   - Locking klevu/module-msi (1.0.0)
#25 9.861   - Locking klevu/module-productsearch (2.7.4)
#25 9.861   - Locking klevu/module-registry (1.1.0)

(dev stability)

**Comment by klevu** [comment]
Thanks for reporting that issue. You're right we'll get that updated.
For now, can you add `"klevu/module-metadata": "1.2.3"` to your composer to stop it from installing `1.3.0`

**Comment by grojanteam** [comment]
yep, I've done it, waiting for global fix :)

**Comment by klevu** [comment]
I have released a new version of metadata `1.3.1` with restrictions on`klevu/module-productsearch`: `^2.10.0|^3.2.0`. I have unpublished the version released earlier today `1.3.0`. 
`1.3.0` should no longer install on older versions and will be restricted to `1.2.3`.

I have also requested that the `1.3.0` be pulled from Magento Marketplace, unfortunately that is not something we can do ourselves. 
If you find that `1.3.0` is still being installed please add  `"exclude": ["klevu/*"]` to your repositories section of composer json to force it to install Klevu modules via packagist
```json
"repositories": [
    {
        "type": "composer",
        "url": "https://repo.magento.com/",
        "exclude": ["klevu/*"]
    }
]
```

**Comment by grojanteam** [comment]
thank you, looks ok for me now, I think you can close now
